### DISCUSSION TITLE
Inverter Powerlimit zusammenfassen

### DISCUSSION THREAD

**Discussion by thias81** [tbnobody/OpenDTU#1374] [Ideas]
Aktuell ist es bei mir so, dass ich für jeden einzelnen Inverter das Powerlimit setzen kann.
Meine Idee wäre folgende.
Zusätzlich die Möglichkeit schaffen, das Powerlimit von allen angeschlossene Invertern zusammen zu fassen und mit einer Eingabe alle Inverter zu regeln.
Dem nach könnte man eine Nulleinspeisung noch einfacher realisieren.

**Comment by Solarteur** [comment]
So etwas wurde im "Schwesterprojekt" schon diskutiert:
https://github.com/lumapu/ahoy/issues/781
